src/test/java/BaseTest.java

```diff
@@ -0,0 +1,26 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+import java.util.concurrent.TimeUnit;
+
+
+public class BaseTest {
+  WebDriver driver;
+  WebDriverWait driverWait;
```

Comment:
Class variables mustn't be public. They should be private or protected

Reply:
done